### PR TITLE
bench: Add benchmark for local storage

### PR DESCRIPTION
**Pull request by uran0sH**
Signed-off-by: Bosco Huang <huangwenyuu@outlook.com>

### CODE REVIEW

File: src/server/storage_engine/default_engine.rs [hunk 1]
```diff
@@ -237,6 +237,7 @@ impl StorageEngine for DefaultEngine {
             write_size,
             data.len()
         );
+        nix::unistd::close(fd)?;
```
Comment (luanshaotong) [inline src/server/storage_engine/default_engine.rs:240]: Also add this for read_file.

**Comment by luanshaotong** [comment]
Maybe buffer for `fd` will speed up `read` and `write`. Never mind, this is a good commit.

**Comment by uran0sH** [comment]
> Maybe buffer for `fd` will speed up `read` and `write`. Never mind, this is a good commit.

The OS doesn't allow open many fds at the same time.

File: src/server/storage_engine/default_engine.rs (hunk 1 above)
Comment (uran0sH) [reply src/server/storage_engine/default_engine.rs:241]: Done

**Review by luanshaotong — APPROVED** [review]

**Pull request merged**